game/forms_avatar.py

```
@@ -0,0 +1,45 @@
+        file_name = getattr(f, "name", "") or "avatar"
+        content_type = getattr(f, "content_type", "") or ""
+        size_bytes = getattr(f, "size", 0)
+
+        # Read bytes once; validations use magic bytes + Pillow verify.
+        file_bytes = f.read()
+
+        # Ensure pointer reset for any downstream consumers.
+        try:
+            f.seek(0)
+        except Exception:
+            pass
```

Ruff (0.15.14)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)